### ISSUE TITLE
Department selection unreachable by keyboard navigation

### ISSUE THREAD

**Issue by will0684** [gcdevops/HRWhiteListing#93]
Department menu is not accessible through keyboard navigation.

<img width="1099" alt="Screen Shot 2020-04-01 at 4 06 24 PM" src="https://user-images.githubusercontent.com/31868510/78181817-f96bcd80-7432-11ea-952e-dec80c604e1d.png">

You can read about this WCAG2.0 success criterion here:

https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html

**Comment by shawnthompson** [comment]
Definitely a make or break issue. First thing we check is keyboard.
